autoreconf fails because steghide do not provide AUTHORS,
NEWS and ChangeLog files

Use 'foreign' keyword to allow automake runs whithout these files.

--- steghide-0.5.1/configure.in
+++ steghide-0.5.1/configure.in
@@ -1,7 +1,8 @@
 dnl Process this file with autoconf to produce a configure script.
-AC_INIT(src/main.cc)
+AC_INIT([steghide], [0.5.1])
+AC_CONFIG_SRCDIR([src/main.cc])
 AC_CANONICAL_SYSTEM
-AM_INIT_AUTOMAKE(steghide, 0.5.1)
+AM_INIT_AUTOMAKE([foreign])
 AM_CONFIG_HEADER(config.h)
 
 dnl checks for programs.
